fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/data/DataValidatorBuilder.java

```diff
@@ -180,6 +180,28 @@ public DataValidatorBuilder notNull() {
         return this;
     }
 
+    public DataValidatorBuilder notNullCheckbox() { 
```

Comment:
This method is not required. The login in LoanRescheduleRequestDataValidator is not correct. Please correct it.